packages/at_commons/lib/src/verb/syntax.dart

```diff
@@ -130,6 +130,8 @@ class VerbSyntax {
       r'^enroll:(?<operation>request|approve|deny|revoke|list)'
       // Positive lookbehind - Matches the value of operation named group
       //  To ignore ":" if operation is list
+      //  To be precise, this is actually to require ":"
+      //  if operation is request|approve|deny|revoke
```

Comment:
I didn't mean for you to literally transcribe what I wrote :-)

How about, `Look-behind to require ':' if operations is request|approve|deny|revoke`

On looking at this again, shouldn't we also require params when operation is request|approve|deny|revoke ?

An alternative regex would be to require `enroll:` then require either `list$` or `(request|approve|deny|revoke):(params fragment)$` which might be simpler to maintain

Reply:
I think you need to write some unit tests